### ISSUE TITLE
marketing pages missing h1s

### ISSUE THREAD

**Issue by susanev** [pulumi/pulumi-hugo#2485]
h1s are important for both accessibility and seo
every page should have exactly 1 h1

this is gonna need some ux design work, @susanev is happy to give feedback

<img width="419" alt="Screen Shot 2023-02-10 at 6 48 28 AM" src="https://user-images.githubusercontent.com/5489125/218121070-a430cabd-243a-4b61-a86b-e6cf26ded609.png">

**Comment by SaraDPH** [comment]
I looked at the layout for the challenge, but I didn't see where the header is so here goes:

**Startup in box**
- https://www.pulumi.com/challenge/startup-in-a-box/

Change "Welcome to Pulumi Challenge" to "**Pulumi Challenge: Startup in a box**" and make it h1
And remove the other header. All the other challenge pages will follow the same pattern
![image](https://user-images.githubusercontent.com/100384099/218185574-3ff7b621-3af2-4a20-96ef-e6f0035dd36d.png)

**One Quickstart**
- https://www.pulumi.com/challenge/one-quickstart/

Change "Welcome to Pulumi Challenge" to "**Pulumi Challenge: One Quickstart to Rule them All**" and make it h1

**Deployments**
- https://www.pulumi.com/challenge/deployments/

Change "Welcome to Pulumi Challenge" to "**Pulumi Challenge: Deployments**" and make it h1

**Deployments**
- https://www.pulumi.com/challenge/holiday-shopping/

Change "Welcome to the Holiday Shopping Challenge!" to "**Pulumi Challenge: Holiday Shopping**" and make it h1

### Cloud Engineering

I tried to update to h1, but the font size needs to be fixed for the Cloud engineering days. It should remain the same size as it had with h5. I don't know how to fix it. :(
PR: https://github.com/pulumi/pulumi-hugo/pull/2486
